alibi_detect/od/tests/test__knn/test__knn.py

```diff
@@ -24,6 +24,17 @@ def make_knn_detector(k=5, aggregator=None, normalizer=None):
 def test_unfitted_knn_single_score():
     knn_detector = KNN(k=10)
     x = np.array([[0, 10], [0.1, 0]])
+    x_ref = np.random.randn(100, 2)
```

Comment:
Are these tests being added here to sync the knn and mahalanobis tests with those used for the pca detector? Wondering if it might be worth opening a separate PR for that, or renaming this one slightly. Just so reverting this one in the future wouldn't have the unintended effect of changing the knn and mahalanonbis tests.

Reply:
OK fair enough, I don't feel that strongly!